### ISSUE TITLE
add support for stk-600 and new tiny and mega devices

### ISSUE THREAD

**Issue by syddo** [platformio/platform-atmelavr#12]
Hi,

i'd like to add support for the Atmel STK-600 and the new ATMega324PB, ATtiny817 devices also.
how do i add a framework for the native avr-gcc?

**Comment by ivankravets** [comment]
Do we need special toolchain for these boards?

**Comment by syddo** [comment]
No. But the current avr-gcc needs to be updated to the latest release from Atmel.

**Comment by ivankravets** [comment]
Ok, please help us.

The AVR toolchain is located in `~/.platformio/toolchain-atmelavr`. You can replace it with own version. Please keep `package.json` file without changes in that place. Otherwise, PlatformIO Package Manager will not see this package.

If it works, we will update current toolchain with your version. See https://bintray.com/platformio/dl-packages/toolchain-atmelavr#files
